### PR TITLE
Add PyFlames 2021 blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @raukadah @RiteshAgrawal 

Please comment the website links for any communities I've missed. 

Also, http://pune.python.org.in/ and http://bangalore.python.org.in/ are down.

### CODE REVIEW

File: content/blog/2021/pyflames.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## The Communities
+
+Various Python communities across India have collaborated to make these mega meetups a possibility. We are very grateful to all the communities. You can find out more about the communities:
+
+[PyDelhi](https://pydelhi.org/)
+[HydPy](https://www.hydpy.org/)
+[BangPypers]()
+[Python Pune]()
+[ChennaiPy](https://chennaipy.org/)
+[Pyladies](https://pyladies.com/)
```
Comment (raukadah) [inline content/blog/2021/pyflames.md:41]: PyLadies India

File: content/blog/2021/pyflames.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+_12:30 - 12:50_ - Ethical Considerations While Building AI Systems - Anjali Bhavan
+_12:50 - 1:10_ - Build Serverless Python Applications using AWS Chalice - Harsh Bardhan Mishra
+_1:10 - 1:30_ - Python and Flutter application for Colouring and Enhancing Old Photos - Utkarsh Mishra
+
+_Post-event_ - PyCon India Office Hours (AMA regarding PyCon India 2021)
+
+## The Communities
+
+Various Python communities across India have collaborated to make these mega meetups a possibility. We are very grateful to all the communities. You can find out more about the communities:
+
```
Comment (raukadah) [inline content/blog/2021/pyflames.md:35]: Can we put these names alphabetically ?

**Comment by anistark** [comment]
@raukadah Can we update BangPypers link by tomorrow? We can do it in a separate PR.

**Comment by raukadah** [comment]
> @raukadah Can we update BangPypers link by tomorrow? We can do it in a separate PR if this gets merged today.

We can add the twitter or meetup link. Update the new link later.

**Comment by Sc0rpi0n101** [comment]
I've added the meetup link for now. Let me know if you want some other link.

**Comment by anistark** [comment]
Works for me.

**Review by anistark — APPROVED** [review]

**Review by raukadah — APPROVED** [review]

**Pull request merged**